examples/02_first_level_models/plot_fir_model.py

```diff
@@ -0,0 +1,155 @@
+"""Analysis of an fMRI dataset with a Finite Impule Response (FIR) model
+=====================================================================
+
+FIR models are used to estimate the hemodyamic response non-parametrically.
+The example below shows that they're good to do statistical inference
+even on fast event-related fMRI datasets.
+
+Specifically, the so-called 'localizer' dataset proposed by Pinel is
```

Comment:
Maybe replace by "Here, we demonstrate the use of a FIR model with 3 lags, computing 4 contrasts from a single subject dataset from the "Neurospin Localizer". It is a fast event related design: During 5 minutes, 80 events of the following types are presented : ['calculaudio', 'calculvideo'', 'clicDvideo', 'clicGaudio', 'clicGvideo', 'damier_H', 'damier_V', 'phraseaudio', 'phrasevideo']"